include/benchmark/benchmark_api.h

```diff
@@ -489,9 +494,9 @@ class Benchmark {
   // REQUIRES: The function passed to the constructor must accept an arg1.
   Benchmark* Range(int start, int limit);
 
-  // Run this benchmark once for every value in the range [start..limit]
+  // Run this benchmark once for all values in the range [start..limit] with specific step
```

Comment:
It's just git messing up history. This change is already in master, but I merged and caught the commit. I'll commit this patch without history.